api/models/User.js

```diff
@@ -3,50 +3,20 @@ var promise       = require('promised-io/promise');
 var adBox         = require('boxfish-router');
 var colors        = require('colors');
 
+var customLogger = function customLogger() {
```

Comment:
I feel like this should be in another file ( maybe another npm module ? ) so ir can be used in all files across the hook